### PR TITLE
oVirt site redesign

### PR DESCRIPTION
**Pull request by sandrobonazzola**
Fixes issue #1968

Changes proposed in this pull request:

- oVirt site redesign following https://marvelapp.com/prototype/1g9eh6h6/screen/58501084

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @sandrobonazzola 

This pull request needs review by: @lwrigh @matthewcarleton

### CODE REVIEW

**Comment by jasonbrooks** [comment]
The mockup looks nice, we'll need to mod the wordpress theme for the blog...

**Review by duck-rh — APPROVED** [review]
The changes look good. Just added a few remarks.

**Comment by sandrobonazzola** [comment]
I need to mention this is not exactly as it shows up in the mockup, I had not enough time for doing everything and what I did is probably not completely working also on mobile. So if anyone with web development skills would like to help, I would be happy to get patches :-)

**Comment by lwrigh** [comment]
@sandrobonazzola Can you post some screenshots of what the implementation looks like?

**Comment by sandrobonazzola** [comment]
Home:
home

**Comment by sandrobonazzola** [comment]
Community:
Community

**Comment by sandrobonazzola** [comment]
Upgrade guide:
Upgrade Guide

**Comment by sandrobonazzola** [comment]
Download:
Screenshot_2020-11-20 Download

**Comment by sandrobonazzola** [comment]
Case studies:
Case studies

File: data/navigation.yml [hunk 1]
```diff
@@ -1,9 +1,7 @@
 ---
-- Home: "/"
 - Download: /download/
 - Documentation: /documentation/
 - Developers: /develop/
 - Community: /community/
-- Events: /events/
```
[the remark this answers is not in the document]
Comment (sandrobonazzola) [reply data/navigation.yml:7]: events are still inked from community page https://ovirt.org/community/

File: source/_sass/bootstrap-custom.sass [hunk 2]
```diff
@@ -27,9 +27,13 @@ $gray-lighter: lighten(black, 93.5%)
 // -------------------------
 
 //$brand-primary: #619a06
-$brand-primary: #61b60e
-$brand-secondary: #73d216
-$brand-success: #5cb85c
+//$brand-primary: #61b60e
```
[the remark this answers is not in the document]
Comment (sandrobonazzola) [reply source/_sass/bootstrap-custom.sass:30]: I'll remove them

**Comment by lwrigh** [comment]
These changes look really nice! LGTM

**Comment by michalskrivanek** [comment]
nice, LGTM

**Comment by jasonbrooks** [comment]
> The mockup looks nice, we'll need to mod the wordpress theme for the blog...

I carried these changes across to the WP blog theme.

**Comment by sandrobonazzola** [comment]
Thanks @jasonbrooks !

**Pull request merged**